__tests__/app.test.js

```diff
@@ -300,4 +300,31 @@ describe('/api/articles/:article_id/comments', () => {
         expect(res.body.msg).toBe('URL NOT FOUND')
       })
     })
-  })
\ No newline at end of file
+  })
+
+  describe("GET api/users", () => {
+    test("GET /api/users should return status code 200", () => {
+      return request(app)
+      .get("/api/users")
+      .expect(200)
+    });
+    test('404 error when incorrect URL is used and message URL not found', () =>{
+      return request(app)
+      .get('/api/userss')
+      .expect(404).then((response) => {
+        expect(response.body.msg).toBe('URL NOT FOUND');
+      });
+    })
+    test('Array of correct data is returned', () =>{
+      return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then((res) => {
+        res.body.forEach((user) => {
```

Comment:
when running a .forEach in your test, you _must_ do a length check first. Something like `expect(res.body.length > 0).toBe(true)` _otherwise_ you'll find that you are auto-passing :)